src/sbt-test/sbt-release/doge-integration/test

```diff
@@ -0,0 +1,5 @@
+> release
+> assertArtifactExists library 2.10
```

Comment:
Sure. I originally had most of the default release steps in here, but yeah, if version and name and the rest of the components of the artifact path remain invariant, no reason not to just hard code the path in the test and use the built-in `exists`.